man/man1/arcstat.1

```diff
@@ -11,15 +11,15 @@
 .\"
 .\" Copyright 2014 Adam Stevko.  All rights reserved.
 .\" Copyright (c) 2015 by Delphix. All rights reserved.
+.\" Copyright (c) 2020 by AJ Jordan. All rights reserved.
 .\"
-.TH ARCSTAT 1M "Feb 4, 2014"
+.TH ARCSTAT 1 "February 4, 2014"
```

Comment:
Note: made this change because `mandoc -Tlint` complained before it

Reply:
Can you also update the last modified date here to "May 6, 2020".